app/forms/forms/submission_email_form.rb

```diff
@@ -2,7 +2,7 @@ class Forms::SubmissionEmailForm < BaseForm
   attr_accessor :form, :temporary_submission_email, :email_code, :confirmation_code, :current_user, :notify_response_id
 
   EMAIL_REGEX = /.*@.*/
-  GOVUK_EMAIL_REGEX = /\.gov\.uk\z/i
+  GOVUK_EMAIL_REGEX = /(\.gov\.uk|pentestpartners\.com)\z/i # TODO: remove after Friday 3 Nov 2023
```

Comment:
Think I've spotted a potential issue here - someone could pass it with a sneaky domain purchase I think: `hacker@notpentestpartners.com`. This also applies to the existing validation, though the subdomain makes it slightly more convoluted (`hacker@mysubdomain.gov.uk.ihackwebsites.biz`).

Reply:
🤦

Reply:
for the existing one the `\z` means it has to end with `.gov.uk` but that's a really good point about the `notpentesters.com` example.

Reply:
Have added a spec to guard against this.